### DISCUSSION TITLE
What does it mean to bind two adaptors?

### DISCUSSION THREAD

**Discussion by bhboyle** [RaspAP/raspap-webgui#1010] [Q&A]
Hi there. 

Im trying to follow your docs to setup Raspap with two wifi adaptors and step one says...
"Bind the two adapters with the interface names wlan0 and wlan1."

What does this mean. I am not able to find a how to anywhere that speaks to how this is done.

Thank you.

**Comment by billz** [comment]
Binding in this context simply means enabling communication on a particular network interface. For example, most of the newer RPis have an onboard wireless adapter that is bound to `wlan0`. 

If you want to add a second wireless interface, I strongly suggest a [USB WiFi adapter that uses in-kernel drivers](https://docs.raspap.com/faq/#adapters). These adapters work without installing drivers (ie., "plug and play") and in most cases will automatically bind to the `wlan1` interface when connected. Once this is done it will be available in RaspAP's interface.

**Reply by bhboyle** [reply]
Ah thats why I couldnt find a way to do it. I understand now thank you. Everything I found was about bridging.

I did see the notes about the external card needing to have kernal drivers. I just ordered one and hope that is its one that does have a kernal driver. It says it does but sometime the advertisers are not totaly honest in thier advertizing.

Cheers.
